package.json

```diff
@@ -27,7 +27,8 @@
     "express": "^4.16.1",
     "install": "^0.10.1",
     "mongodb": "^2.2.31",
-    "npm": "^5.4.2"
+    "npm": "^5.4.2",
+    "uuid": "^3.1.0"
```

Comment:
`uuid` generiert einzigartige IDs (Universally Unique Identifier), in diesem Fall für die hochgeladenen Bilder. Eigentlich mit Kanonen auf Spatzen geschossen, aber andererseits müssen wir uns dann keine Gedanken über mögliche ID-Clashes machen, und das kostet performancemäßig nicht viel.